### ISSUE TITLE
IE11 compatibility

### ISSUE THREAD

**Issue by ascheink** [nprapps/pym.js#63]
Since upgrading to the latest version of Pym, we've been getting reports that it's not resizing on load from IE11 users. We haven't been able to reproduce it just yet, but I'm wondering if anyone else has seen this issue.

**Comment by basistom** [comment]
Hey, we are getting problems with ie11 users too. neither it calculates the iframe height initial nor changes it after resizing the content: http://goo.gl/HNqcp4

Any ideas?

**Comment by onyxfish** [comment]
@basistom

**Comment by onyxfish** [comment]
@basistom I took a look at this. I'm entirely certain what's happening, however, after refreshing the page several times I did see a partial iframe on one load, so I'm pretty sure it's a race-condition of some sort. Have you tried wrapping your `pym.Child` in a `$.load()`?

I see you are using `polling` which logically should make using load unnecessary, however, I think it's the first thing worth trying.
